modules/javafx.graphics/src/main/java/com/sun/javafx/css/StyleManager.java

```diff
@@ -1271,8 +1271,7 @@ public void setUserAgentStylesheets(List<String> urls) {
                     if (fname == null || fname.isEmpty()) break;
 
                     StylesheetContainer container = platformUserAgentStylesheetContainers.get(n);
-                    // assignment in this conditional is intentional!
-                    if(isSame = fname.equals(container.fname)) {
+                    if((isSame = fname.equals(container.fname)) == true) {
```

Comment:
Minor: since you are updating this line, maybe add a space after the `if`? (although since it is preexisting, it's ok to leave it as-is)